src/Command/TranskribusAuthCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class TranskribusAuthCommand extends Command
+{
+
+    private $client;
+
+    public function __construct(HttpClientInterface $client)
+    {
+        parent::__construct();
+        $this->client = $client;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setHelp('This command allows you to retrieve and store an access token for testing the Trankribus engine');
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $helper = $this->getHelper('question');
+        $io = new SymfonyStyle($input, $output);
+
+        $userNameQuestion = new Question('Enter username: ');
+        $userName = $helper->ask($input, $output, $userNameQuestion);
+
+        $passwordQuestion = new Question('Enter password: ');
+        $password = $helper->ask($input, $output, $passwordQuestion);
+
+        $clientId = $_ENV['CLIENT_ID'];
```

Comment:
Client ID should be injected in the constructor, with something like the following in `services.yaml`:

```yaml
        arguments:
            $clientID: '%env(APP_TRANSKRIBUS_CLIENT_ID)%'
```